src/ir/datalog/input_relation_fact_test.cc

```diff
@@ -70,11 +76,13 @@ class OneOfEachFact
 };
 
 TEST(OneOfEachFactTest, OneOfEachFactTest) {
```

Comment:
@markww do we have enough test coverage?

Reply:
I think we have enough unit test coverage, but I think we are missing integration test coverage. That is, these strings aren't consumed by Souffle at this point, so we can't notice issues with Souffle misparsing them. I think some integration tests having Souffle parse the results may be in order soon.

Reply:
Yes, I think that sounds right.